lib/user-db.js

```diff
@@ -1,5 +1,6 @@
 const db = require('../db'),
-      async = require('async');
+      async = require('async'),
+      DB_NAME_BLACKLIST = /[^a-z0-9_\$\(\)\+\-/]/g;
```

Comment:
It was only the hyphen that needed escaping because it interpreted `+-/` as being a range. I've moved the hyphen to the end of the group and it just works with no escaping. The unit test fails if this gets messed up so we should be safe from regressions.